tests/testthat/custom_tests/test-regional-dataset.R

```diff
@@ -40,11 +40,16 @@ test_regional_dataset <- function(source, level, download = FALSE) {
   if (download) {
     test_that(paste0(data_name, " downloads sucessfully"), {
       region$download()
-      expect_s3_class(region$data$raw, "data.frame")
-      expect_true(nrow(region$data$raw) > 0)
-      expect_true(ncol(region$data$raw) >= 2)
+      purrr::walk(region$data$raw, function(data) {
```

Comment:
nice nice. Have you tested this all with `options(testDownload = TRUE)` needs to be done before merging.

Reply:
Yeah just did it again with the changes for the latest commit